python/.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
```

Comment:
What is this file for? Can we remove it?

Reply:
Actually, I think enforcing coding standards is a good thing! If we keep the file, can we have CI run the check using this tool?

Reply:
I've reinstated the file.

I'll take a look, but I know this is aimed at running checks before a git commit.

I have a feeling I saw something about a pre-commit-ci that I think works with github actions, which would be great.

Reply:
Looks like the best approach is using https://pre-commit.ci/ which is available on the github marketplace, for what I can see, and is free for open source projects... A nice to have, perhaps?

Reply:
I guess @aslakhellesoy suggested to check styling/standards as part of the CI, but not using a pre-commit hook

Like it is done for example in javascript with eslint, or ruby with rubocop: those are part of the CI beside the tests